server/api/codeeditor.js

```diff
@@ -0,0 +1,36 @@
+const router = require('express').Router()
+const db = require('../db/db')
+const util = require('util')
+
+const TestTable = db.define('testtable', {
+  body: db.Sequelize.TEXT
+})
+
+module.exports = router
+
+router.get('/', (req, res, next) => {
+  TestTable.findByPk(1)
+    .then(s => res.json(s))
+    .catch(next)
+})
+
+router.post('/', (req, res, next) => {
+  TestTable.create(req.body)
+    .then(s => res.json(s))
+    .catch(next)
+})
+
+router.post('/runcode', (req, res, next) => {
+  let result = ''
+  const myConsoleLog = (...args) => {
```

Comment:
where is myConsoleLog called?

Reply:
See line 30. That's replacing any console.logs in the code the user has written with myConsoleLog. This is needed b/c other the eval function would just print out everything in the console.logs on the server but wouldn't store it in a variable that we could send back to the client